pyproject.toml

```diff
@@ -2,7 +2,18 @@
 name = "lookit-api"
 version = "0.1.0"
 description = ""
-authors = [""]
+authors = ["MIT <mit@mit.com>"]
```

Comment:
Can this just be MIT? (mit.com isn't a domain that makes sense to use as a placeholder - we could always use lookit@mit.edu if we want a durable-ish reference that actually goes to us)

Reply:
I meant for it to be "...mit.edu".  I'll update it to what you suggested above.